aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -678,18 +678,6 @@ static void frexp_kernel(TensorIteratorBase& iter) {
   }                                                                                              \
   REGISTER_DISPATCH(op##_stub, &op##_kernel)
 
-  #define IMPLEMENT_COMPLEX_STRUCTURED_KERNEL(op)                                                \
```

Comment:
why does this have to go?

Reply:
It doesn't, it just turns out it's identical to the macro above.

Reply:
ah is the `iter.cast_outputs()` on 689 redundant

Reply:
Hmm, actually, no, that might be a bug...

Reply:
Ah yeah, that was added in this PR: https://github.com/pytorch/pytorch/pull/52277. I think it's fine to remove the old macro now that all unary ops have been ported. The old un-structured kernels were written to call `cast_outputs()` directly for some reason, which is why it was added to the macro for structured kernels